eva-pipeline/src/test/java/embl/ebi/variation/eva/pipeline/jobs/AnnotationConfig.java

```diff
@@ -0,0 +1,26 @@
+package embl.ebi.variation.eva.pipeline.jobs;
+
+import org.springframework.batch.test.JobLauncherTestUtils;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.context.support.PropertySourcesPlaceholderConfigurer;
+
+/**
+ * @author Diego Poggioli
+ *
+ * Configuration init for Annotation Job
+ */
+@Configuration
+@PropertySource({"annotation.properties"})
+public class AnnotationConfig {
```

Comment:
It is loading the ApplicationContext with props.. otherwise you get:
` java.lang.IllegalStateException: Failed to load ApplicationContext
`